docs/source/index.rst

```diff
@@ -6,6 +6,23 @@
 Welcome to glidertest's documentation!
 ======================================
 
+Glidertest is a Python package aiming to diagnose possible issues with your glider dataset. The package takes as input data from any glider in OG1 format. 
```

Comment:
How about some links like this ` OG1 format <https://github.com/OceanGlidersCommunity/OG-format-user-manual>`_